app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -131,26 +131,27 @@ export default {
         { text: 'Submission Date', align: 'start', value: 'date' },
         { text: 'Submitter', align: 'start', value: 'submitter' },
       ];
+
+      // If status flow enabled add that column
       if (this.showStatus) {
         headers.splice(3, 0, {
           text: 'Status',
           align: 'start',
           value: 'status',
         });
       }
-      if (
-        this.userFormPreferences &&
-        this.userFormPreferences.preferences &&
-        this.userFormPreferences.preferences.columnList
-      ) {
-        this.userFormPreferences.preferences.columnList.forEach((col) => {
-          headers.push({
-            text: col,
-            align: 'end',
-            value: col,
-          });
+
+      // Add any custom columns if the user has them
+      const clm = 25;
```

Comment:
May be better to name this something like maxHeaderLength instead of clm for readability purposes.

Reply:
Fixed